### ISSUE TITLE
Font Awesome classes

### ISSUE THREAD

**Issue by FatBoyXPC** [cubing/ccm#208]
I would like to be able to access the font-awesome in our LESS. Currently we can't due to the load order of meteor and such, but if we could do something like we're doing with bootstrap, that would be awesome. I'm not a big fan of this: https://github.com/FatBoyXPC/ccm/commit/45586b853eac5587aaa0c0815533908049ff3281#diff-91f8ed62a3ca96a901b8031d6580b5c7R46

**Comment by jfly** [comment]
Looks like we're using [linto:fontawesome](https://github.com/HedCET/fontawesome), which doesn't look like it has any LESS in it. Maybe the easiest way of doing this would be to copy the fontawesome LESS into CCM ourselves?

**Comment by FatBoyXPC** [comment]
Yeah, that's what I was thinking. We still need to make it available to our importable LESS, though, which means it can't be in a package.

**Comment by FatBoyXPC** [comment]
What about this? https://github.com/lgollut/meteor-font-awesome-less I would definitely prefer to have something like our bootstrap import LESS package.

**Comment by jfly** [comment]
Looks good to me, if you can get it to work =)

**Comment by FatBoyXPC** [comment]
Bah, that package (and the other one I found) are for Meteor 0.7 or 0.8, and 0.9 changed things drastically.

I'll look for other packages, or possibly just create something similar to our bootstrap less package.
